docker-compose.yml

```diff
@@ -96,6 +96,8 @@ services:
     image: adr.datapusher:${IMAGE_TAG}
     build:
       context: ./submodules/datapusher
+    environment:
+      - DATAPUSHER_MAX_CONTENT_LENGTH=${DATAPUSHER_MAX_CONTENT_LENGTH}
```

Comment:
Writing here only:
```
- DATAPUSHER_MAX_CONTENT_LENGTH
```
will be enough as it tells docker-compose to pass the mentioned environment with the same value as exists on the host.